packages/eas-cli/src/submit/ArchiveSource.ts

```diff
@@ -304,42 +305,36 @@ async function handleBuildListSourceAsync(
 function formatBuildChoice(build: BuildFragment, expiryDate: Date): prompts.Choice {
   const {
     id,
-    platform,
     updatedAt,
-    appVersion,
-    sdkVersion,
     runtimeVersion,
     buildProfile,
-    appBuildVersion,
     releaseChannel,
-    initiatingActor,
+    gitCommitHash,
+    gitCommitMessage,
   } = build;
 
   const formatValue = (field?: string | null): string =>
     field ? chalk.bold(field) : chalk.dim('Unknown');
 
   const buildDate = new Date(updatedAt);
-  const maybeRuntimeVersion = runtimeVersion ? `Runtime: ${formatValue(runtimeVersion)}` : null;
-  const maybeSdkVersion = sdkVersion ? `SDK: ${formatValue(sdkVersion)}` : null;
-  const appBuildVersionString = `${
-    platform === AppPlatform.Android ? 'Version code' : 'Build number'
-  }: ${formatValue(appBuildVersion)}`;
-
-  const title = [
-    `ID: ${chalk.dim(id)}, Finished at: ${chalk.bold(buildDate.toLocaleString())}`,
-    [
-      `\tApp version: ${formatValue(appVersion)}, ${appBuildVersionString}`,
```

Comment:
Why did you remove it?

Reply:
Jon (who is the issue author), suggested that only these fields should be displayed:

<img width="857" alt="Screenshot 2022-11-10 at 11 26 01" src="https://user-images.githubusercontent.com/55145344/201066633-eb3c6cd5-3c9d-417b-a6a9-b2d9b3a9e272.png">